### ISSUE TITLE
dpkg-reconfigure fails because default.settings.php is already renamed

### ISSUE THREAD

**Issue by Dave-McCraw** [Dave-McCraw/pkg-emoncms#1]
From [OpenEnergyMonitor forum](http://openenergymonitor.org/emon/node/4067/19802), looks like the postinst script is falling over because default.settings.php has already been renamed to settings.php on the initial installation : 

```
Replacing existing emoncms config as follows:
- MySQL configuration   : root connecting to emoncms db on localhost
- SMTP configuration    :  connecting to  to send as emoncms (noreply) (emoncms-noreply@localhost)
- Allow password resets :
mv: cannot stat `/usr/share/emoncms/www/default.settings.php': No such file or directory
Verifying feed directories exist and have the correct ownership...
- /var/lib/phpfiwa
- /var/lib/phpfina
- /var/lib/phptimeseries
- /var/lib/timestore
Attempting to create 'emoncms' mysql database (with the 'root' user)...
ERROR 1045 (28000): Access denied for user 'root'@'localhost' (using password: YES)
- Failed (check username and password are correct, using dpkg-reconfigure. Check grants allow table creation, etc...)
```

**Comment by Dave-McCraw** [comment]
I'm not sure what the proper mechanism is for taking `default.settings.php` from [emoncms/emoncms](https://github.com/emoncms/emoncms) (upstream) and ending up with a populated `settings.php` file after debian packaging and installation.

One simple-ish option is just to rename the file to `settings.php` in this repository and maintain it using git (it should track upstream changes even after renaming). That would also allow the upstream `default.settings.php` file to change away from my debconf / postinst placeholders, which I now think is clearly inappropriate since we're putting the debian stuff in this package.

Don't know if @Jerome-github may have any thoughts?

**Comment by lafrech** [comment]
I don't know. From my Debian experience, this is what I'd expect as a user:
- settings.php is installed as settings.php and I need to adapt it to my needs manually, ideally with default values that make sense
- When a new version comes, I'm warned about a conflict and need to deal with it manually. Hopefully, this does not happen too often. I don't know of any better way and if it existed, our lives would be easier.
- DB settings could be moved to another file, that could be filled by a simple post-install wizard, the kind you get when using dpkg-reconfigure emoncms. This move does not need to be done upstream. It can be just a php "import". Anyway, this is not mandatory at all.

Moving default.settings.php to settings.php should let git understand the renaming and follow upstream changes.

**Comment by Dave-McCraw** [comment]
Fixed as 8.0.3-3, closed by a9ee940cfb608e642caa5f0b3d839a2feeab3208
